pkg/proxy/proxy.go

```diff
@@ -199,14 +240,6 @@ func (pm *Manager) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	// If the legacy auth provider cookie exists, delete it.
-	http.SetCookie(w, &http.Cookie{
-		Name:   ObotAuthProviderQueryParam,
-		Value:  "",
-		Path:   "/",
-		MaxAge: -1,
-	})
```

Comment:
It's been long enough, that there should no longer be any of these out there. They won't do or interfere with anything anyway.